<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyForm</class>
 <widget class="QWidget" name="MyForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="functionalArea" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <!-- Removed: newButton -->
       <!-- <widget class="QPushButton" name="newButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>New</string>
        </property>
       </widget> -->
       <!-- Removed: openButton -->
       <!-- <widget class="QPushButton" name="openButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Open</string>
        </property>
       </widget> -->
       <!-- Removed: saveButton -->
       <!-- <widget class="QPushButton" name="saveButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget> -->
       <!-- Removed: saveAsButton -->
       <!-- <widget class="QPushButton" name="saveAsButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Save As</string>
        </property>
       </widget> -->
       <!-- Removed: undoButton -->
       <!-- <widget class="QPushButton" name="undoButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Undo</string>
        </property>
       </widget> -->
        <!-- Removed: redoButton -->
       <!-- <widget class="QPushButton" name="redoButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>10</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Redo</string>
        </property>
       </widget> -->
       <widget class="QLabel" name="statusLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>500</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ready</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>65</y>
          <width>500</width>
          <height>20</height>
         </rect>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="headerLabel">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>100</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Map Controls:</string>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </widget>
       <!-- Removed: loadMapButton -->
       <!-- <widget class="QPushButton" name="loadMapButton">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Map</string>
        </property>
       </widget> -->
       <!-- Removed: zoomInButton -->
       <!-- <widget class="QPushButton" name="zoomInButton">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Zoom In</string>
        </property>
       </widget> -->
       <!-- Removed: zoomOutButton -->
       <!-- <widget class="QPushButton" name="zoomOutButton">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Zoom Out</string>
        </property>
       </widget> -->
       <!-- Removed: panButton -->
       <!-- <widget class="QPushButton" name="panButton">
        <property name="geometry">
         <rect>
          <x>280</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Pan</string>
        </property>
       </widget> -->
       <!-- Removed: loadTileMapButton -->
       <!-- <widget class="QPushButton" name="loadTileMapButton">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>120</y>
          <width>100</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Load Tile Map</string>
        </property>
       </widget> -->
       <!-- Removed: zoomInTileMapButton -->
       <!-- <widget class="QPushButton" name="zoomInTileMapButton">
        <property name="geometry">
         <rect>
          <x>480</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Zoom In TM</string>
        </property>
       </widget> -->
       <!-- Removed: zoomOutTileMapButton -->
       <!-- <widget class="QPushButton" name="zoomOutTileMapButton">
        <property name="geometry">
         <rect>
          <x>570</x>
          <y>120</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>Zoom Out TM</string>
        </property>
       </widget> -->
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="handleWidth">
        <number>4</number>
       </property>
       <widget class="QListView" name="listView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots/>
</ui>